install.sh

```diff
@@ -4,9 +4,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+if vim --version | grep -q +python3; then
```

Comment:
This autodetection will malfunction if e.g. users are actually using MacVim, with a different binary name, and its Python support differs from whatever "vim" is. I don't think dealing with this in install.sh is really necessary, but it might be nice to:

1. Call out that this is happening in the README.

2. Only do this detection if PY3 isn't already defined, allowing users to provide an override.